app/src/main/res/layout/ar_layout.xml

```diff
@@ -16,12 +16,13 @@
         android:layout_marginEnd="@dimen/activity_horizontal_margin"
         android:layout_marginBottom="@dimen/activity_vertical_margin"
         app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintTop_toTopOf="@+id/constraintLayout"
+        app:layout_constraintTop_toTopOf="@id/constraintLayout2"
```

Comment:
I noticed there's a +id here, not sure why since it refers an existing name and not adding a new one.